GitUI/Resources/ChangeLog.md

```diff
@@ -1,6 +1,31 @@
 Changelog
 =========
 
+### [Version 2.51.01] (11 Mar 2018)
+
+#### Features:
+* TaskCanceledException for ShowSelectedFileDiff() - Issue [4570]
```

Comment:
Should not be in change log 2.51->2.51.00

Reply:
Sorry, I don't understand... Could you clarify?